Gemfile

```diff
@@ -0,0 +1,15 @@
+ruby '2.2.2'
```

Comment:
I think gems aren't supposed to declare Ruby versions, leaving that to the including apps. (Many are intentionally tested against multiple versions.)